lib/MusicBrainz/Server/Controller/Artist.pm

```diff
@@ -146,9 +146,12 @@ after 'load' => sub
         }
     }
 
-    $c->model('ArtistType')->load($artist, map { $_->target } @{ $artist->relationships_by_type('artist') });
-    $c->model('Gender')->load($artist);
-    $c->model('Area')->load($artist);
+    my $lang = $c->stash->{current_language} // 'en';
```

Comment:
I'm pretty clueless about how this `c` (context?) stash stuff works. I'm passing the language around since `current_language` didn't seem to be available in `load_related_info` and the indexed search code, but please let me know if there's an easier way to get at it there.